www/js/control.js

```diff
@@ -0,0 +1,431 @@
+        var accuracyActions = [];
+        for (name in $scope.settings.collect.accuracyOptions) {
+            accuracyActions.push({text: name, value: $scope.settings.collect.accuracyOptions[name]});
+        }
+        $ionicActionSheet.show({
+            buttons: accuracyActions,
+            titleText: "Select accuracy",
+            cancelText: "Cancel",
+            buttonClicked: function(index, button) {
+                $scope.settings.collect.new_config.accuracy = button.value;
+                return true;
+            }
+        });
+    };
+    $scope.setSyncInterval = function() {
+        var syncIntervalActions = [];
+        syncIntervalActions.push({text: "1 min", value: 60});
+        syncIntervalActions.push({text: "10 min", value: 10 * 60});
+        syncIntervalActions.push({text: "30 min", value: 30 * 60});
+        syncIntervalActions.push({text: "1 hr", value: 60 * 60});
+        $ionicActionSheet.show({
+            buttons: syncIntervalActions,
+            titleText: "Select sync interval",
+            cancelText: "Cancel",
+            buttonClicked: function(index, button) {
+                $scope.settings.sync.new_config.sync_interval = button.value;
+                return true;
+            }
+        });
+    };
+
+    $scope.isAndroid = function() {
+        return ionic.Platform.isAndroid();
+    }
+
+    $scope.isIOS = function() {
+        return ionic.Platform.isIOS();
+    }
+
+    $scope.refreshScreen();
+    $ionicPopover.fromTemplateUrl('templates/control/main-collect-settings.html', {
+        scope: $scope
+    }).then(function(popover) {
+        $scope.collectSettingsPopup = popover;
+    });
+    $ionicPopover.fromTemplateUrl('templates/control/main-sync-settings.html', {
+        scope: $scope
+    }).then(function(popover) {
+        $scope.syncSettingsPopup = popover;
+    });
+    $scope.trackingOn = function() {
+        return $scope.settings.collect.state != "STATE_TRACKING_STOPPED";
+    }
+    $scope.userStartStopTracking = function() {
+        if ($scope.startStopBtnToggle){
+            $scope.forceTransition('STOP_TRACKING');
+            $scope.startStopBtnToggle = false;
+        } else {
+            $scope.forceTransition('START_TRACKING');
+            $scope.startStopBtnToggle = true;
+        }
+    }
+    $scope.startStopBtnToggle = $scope.trackingOn();
+    $scope.getExpandButtonClass = function() {
+        return ($scope.expanded)? "icon ion-ios-arrow-up" : "icon ion-ios-arrow-down";
+    }
+    $scope.getUserDataExpandButtonClass = function() {
+        return ($scope.dataExpanded)? "icon ion-ios-arrow-up" : "icon ion-ios-arrow-down";
+    }
+    $scope.eraseUserData = function() {
+        UserCalorieData.delete();
+        $ionicPopup.alert({template: 'User data erased.'});
+
+    }
+    $scope.parseState = function(state) {
+        if (state) {
+            return state.substring(6);
+        }
+    }
+    $scope.toggleCollection = function() {
```

Comment:
Can we rename this? It sounds like we are _toggling_ (turning on and off) collection.